charts/gatekeeper/README.md

```diff
@@ -166,6 +166,7 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | controllerManager.podSecurityContext          | Security context on pod level for controller manager                                                                                                                                                                                                          | {fsGroup: 999, suplementalGroups: [999]}                                    |
 | controllerManager.exemptNamespaces            | The exact namespaces to exempt by the admission webhook                                                                                                                                                                                             | `[]`                                                                      |
 | controllerManager.exemptNamespacePrefixes     | The namespace prefixes to exempt by the admission webhook                                                                                                                                                                                           | `[]`                                                                      |
+| controllerManager.exemptNamespaceSuffixes     | The namespace suffixes to exempt by the admission webhook                                                                                                                                                                                           | `[]`                                                                      |
```

Comment:
For helm chart updates, PTAL https://open-policy-agent.github.io/gatekeeper/website/docs/help#contributing-to-helm-chart and make this update in https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/static/README.md instead then run `make manifests` to generate changes in the [manifest_staging](https://github.com/open-policy-agent/gatekeeper/tree/master/manifest_staging) directory.